modules/sno-du-reducing-resource-usage-with-cluster-monitoring.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-reducing-resource-usage-with-cluster-monitoring_{context}"]
+= Reducing resource usage with cluster monitoring
+
+
```

Comment:
@imiller0 This is one of the new topics. Please provide an intro sentence here - basically, why they are using this procedure.

Reply:
added.